### DISCUSSION TITLE
PDF Support

### DISCUSSION THREAD

**Discussion by Kira-Kitsune** [Brooooooklyn/canvas#541] [General]
Hello,
I was wondering, what would the ETA for PDF Support? I need it for a project and alternatives of this repo doesn't work for my project.
Thanks,
Kira

**Comment by Brooooooklyn** [comment]
@Kira-Kitsune hey, I'm planning to implement PDF support recently

**Reply by Kira-Kitsune** [reply]
Would there be an estimate when it will be implemented? No rush ofc, great work btw :3

**Comment by programmer24601** [comment]
@Brooooooklyn any updates on pdf support?
